client/src/pages/admin/courses.tsx

```diff
@@ -241,12 +244,12 @@ function Page() {
           label="Minimum Students per Mentor"
           rules={[{ min: 1, type: 'integer', message: 'Ensure that the input, if provided, is a positive integer.' }]}
         >
-          <InputNumber step={1} defaultValue={2} />
+          <InputNumber step={1} />
```

Comment:
why did you remove the default value here? Does it cause any problems?

Reply:
@valerydluski yes, it conflicts with input mode (controlled)  as per warning. I moved to initial values.